kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/server/ServerInterceptor.kt

```diff
@@ -0,0 +1,35 @@
+package xyz.block.ftl.server
+
+import io.grpc.*
+import io.grpc.ServerInterceptor
+
+const val ftlVerbHeader = "FTL-Verb"
+const val ftlRequestIdHeader = "FTL-Request-ID"
```

Comment:
Not for this PR, but I wonder if stuff like this would be good to put in protos so we can generate the same const for all the platforms we plan to support in the future.

Reply:
Protos don't have constants.